coverage_runner.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+# Generate `coverage/lcov.info` file
+flutter test --coverage
+
+# Generate HTML report
+# Note: on macOS you need to have lcov installed on your system (`brew install lcov`) to use this:
+genhtml coverage/lcov.info -o coverage/html
+
+# Open the report
+open coverage/html/index.html
```

Comment:
Here and throughout, can you ensure your source files end with a trailing newline? You can probably set this as an editor default, most style guides agree on a single trailing newline.